CommandHelper/LocalPackages/commands/buffs/genos.ms

```diff
@@ -12,6 +12,8 @@
             'speech': "Master Saitama would be proud.",
             'powers': array(
                 "/buff",
+                "/carryplayer",
+                "/dismountplayer",
                 "/jump 0-8",
                 "/run 0-8",
```

Comment:
Fix Run 0-8 its supposed to be 0-4 since the run can only go up till 4 for Genos.